StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1130CodeFixProvider.cs

```diff
@@ -221,7 +249,14 @@ private static async Task<Document> GetTransformedDocumentAsync(Document documen
 
             var anonymousMethod = (AnonymousMethodExpressionSyntax)syntaxRoot.FindNode(diagnostic.Location.SourceSpan, getInnermostNodeForTie: true);
 
-            var newSyntaxRoot = syntaxRoot.ReplaceNode(anonymousMethod, ReplaceWithLambda(semanticModel, anonymousMethod));
+            var newNode = ReplaceWithLambda(semanticModel, anonymousMethod);
+
+            if (newNode == null)
+            {
+                return document;
```

Comment:
❓ Is this reachable considering the condition added before registering the code fix?

Reply:
I dont think so